custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: v1
```

Comment:
I think if we want to consider new resource model as a default one then it makes sense to name this as adapter.yaml, and rename old file to adapter_old_resource_model.yaml

Reply:
I think we should still treat the legacy one as default, at least for as long as it's default in GKE. I.e. for users that don't care about new model, we shouldn't break if they update their deployment with `kubectl apply -f .../adapter.yaml`

Reply:
Makes sense.